conduct.md

```diff
@@ -72,13 +72,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
+- Create decision that mutually benefit each other (Schedules, Coding). This is to help understand each other's responsibilities and respect on what is possible for both individual interest and the team interest.
+
 - TODO
 - TODO
 
 #### Unacceptable Conduct
 
-- TODO
+- Never informing on others on unattended appointments. This will only create more problems as other team members has to do more work unexpectedly, therefore the work will not be as good as intended to be.
```

Comment:
You might want to rephrase this -- maybe something more like 'Failure to communicate any absences to their team members or Professors.' Additionally, you will want to change the tense of 'has' to 'have'.

Reply:
Was written by another team that already merged to main--content was pulled in when branch was created